app/src/main/java/com/example/memeitupapp/ui/listmemes/model/ListMemesModel.kt

```diff
@@ -0,0 +1,12 @@
+package com.example.memeitupapp.ui.listmemes.model
+
+import com.example.memeitupapp.data.entity.Meme
+import com.example.memeitupapp.data.repository.MemeRepository
+import com.example.memeitupapp.ui.contract.ListMemesContract
+import com.example.memeitupapp.util.Result
+
+class ListMemesModel : ListMemesContract.Model {
+    private val memesRepository = MemeRepository()
```

Comment:
You should pass the repository as a parameter to the model because then if you want to the test the model you can mock the repository. If you have as a private attribute you cant.

Reply:
added, thanks